src/__experimental__/components/checkbox/checkbox.component.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import { validProps } from '../../../utils/ether';
+import FormField from '../form-field';
+import CheckboxStyle from './checkbox.style';
+
+const Checkbox = (props) => {
+  const formFieldProps = () => {
+    const { ...validFormFieldProps } = validProps(
+      { propTypes: Checkbox.propTypes, safeProps: ['fieldHelpInline'], props }
+    );
+
+    validFormFieldProps.reverse = !props.reverse;
+
+    return validFormFieldProps;
+  };
+
+  const inputProps = () => {
+    const { ...validInputProps } = validProps(
+      { propTypes: Checkbox.propTypes, safeProps: ['disabled', 'onChange'], props }
+    );
+
+    validInputProps.type = 'checkbox';
+    delete validInputProps.children;
+    delete validInputProps.fieldHelp;
+    delete validInputProps.labelHelp;
+    return validInputProps;
+  };
+
+  const checkboxSprite = () => {
+    return (
+      <svg
+        width='12' height='10'
+        viewBox='0 0 12 10'
+      >
+        <path
+          d={ 'M.237 6.477A.752.752 0 0 1 .155 5.47l.851-1.092a.63.63 0 0 1 .934-.088l2.697 1.964, '
+            + '4.674-6a.63.63 0 0 1 .933-.088l1.015.917c.28.254.317.703.081 1.005L6.353 8.492a.725.725, '
+            + '0 0 1-.095.16l-.85 1.093a.637.637 0 0 1-.626.244.638.638 0 0 1-.335-.16L.237 6.476z' }
+          className='checkbox-check'
+          fill='#FFFFFF'
+          fillRule='evenodd'
+        />
+      </svg>
+    );
```

Comment:
Maybe could you move it to separate file?

Reply:
I could, but feels a little unnecessary, as it's not going to be used outside of the Checkbox.

Reply:
There is a `className` prop I wonder if it should be transformed to a styled component. If not feel free to resolve this comment.

Reply:
Removed the unnecessary className